lib/delighted/utils.rb

```diff
@@ -67,6 +67,16 @@ def self.wrap_array(object)
         [object]
       end
     end
+
+    def self.full_const_get(name)
```

Comment:
Needed for ruby < 2.1. Afterwards, `const_get` properly recursively looks up constant names if a namespaced class name is provided.